src/test/java/com/hotels/mutantswarm/mutate/LexerMutatorStoreTest.java

```diff
@@ -133,4 +134,26 @@ public void nullStream() {
     lexerStore.getMutatorsFor(position, null);
   }
 
+  @Test
+  public void checkEquals() {
+    Imposter imposter4 = new Imposter(4,"hello");
+    Imposter imposter5 = new Imposter(4,"hello");
+    boolean result = imposter4.equals(imposter5);
+    assertTrue(result);
+  }
+
+  @Test
+  public void checkToString() {
+    Imposter imposter4 = new Imposter(4,"hello");
+    String result = imposter4.toString();
+    assertEquals(result,"Imposter [tokenId=4, text=hello]");
+  }
+
+  @Test
+  public void checkhashCode() {
+    Imposter imposter4 = new Imposter(4,"hello");
+    boolean result = (imposter4.hashCode() == (int)imposter4.hashCode());
```

Comment:
`assertThat(impostert4.hasCode(), is(imposter4.hashCode());` Write it like this for a better error message. Let the test framework do the comparing.
But having said that how is this ever going to be false?

You probably want something like:
```
int expectedHashCode=??;
assertThat(expectedHashCode, is(imposter4.hashCode());
```

Reply:
I tried doing that and it worked until I ran the mvn build and an error appeared when testing the hashcode.

What I've done now is removing the `checkHashCode()` tests and added `assertThat(imposter4.hashCode() == imposter5.hashCode(), is(true))` in the `equalSame()` to test that if the object is the same we get the same hash code.